test/NinjaTools.FluentMockServer.API.Tests/Downstream/EmailExampleTable.cs

```diff
@@ -0,0 +1,24 @@
+using JetBrains.Annotations;
+using TestStack.BDDfy;
+
+namespace NinjaTools.FluentMockServer.API.Tests.Downstream
+{
+    // /// <inheritdoc />
+    // public class EmailExampleTable : ExampleTable
+    // {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4722826)